Lib/pathlib.py

```diff
@@ -585,7 +620,6 @@ def __repr__(self):
 
 class PurePath(object):
     """Base class for manipulating paths without I/O.
-
```

Comment:
I think these really ought to stay here. Are these the whitespaces that were giving you trouble? If so, we need to fix the checks.

Reply:
I'm not sure which exactly whitespace was wrong. I ran functions from  patchcheck.py and after that the checks were passed, but maybe some functions were unnecessary. I'll put those lines back and see checks.

Reply:
Checks have passed with those lines returned.